src/darwin/Framework/CHIP/MTRAttributeUtils.h

```diff
@@ -0,0 +1,25 @@
+/*
+ *
+ *    Copyright (c) 2023 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+
+#import <Foundation/Foundation.h>
+
+#include <lib/core/DataModelTypes.h>
+
+BOOL MTRAttributeIsSpecified(chip::ClusterId aClusterId, chip::AttributeId aAttributeId);
```

Comment:
This could use some documentation.  Is this supposed to be "AttributeIsStandard" (which we can tell by just testing the cluster/attribute value range), or "AttributeIsSomethingWhereWeWouldKnowIfItHadTheCQuality" (which is a totally different situation)?

Reply:
It's the latter.